nautobot/core/api/views.py

```diff
@@ -236,9 +238,9 @@ def restrict_queryset(self, request, *args, **kwargs):
         """
         # Restrict the view's QuerySet to allow only the permitted objects for the given user, if applicable
         if request.user.is_authenticated:
-            action = HTTP_ACTIONS[request.method]
-            if action:
-                self.queryset = self.queryset.restrict(request.user, action)
+            http_action = HTTP_ACTIONS[request.method]
```

Comment:
pylint is not happy with `action` here because it thinks it is a rename for the `@action` decorator